http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/app/CromwellRunnerAppInstall.scala

```diff
@@ -79,6 +86,21 @@ class CromwellRunnerAppInstall[F[_]](config: CromwellRunnerAppConfig,
         AppCreationException(s"Pet not found for user ${params.app.auditInfo.creator}", Some(ctx.traceId))
       )
 
+      leoAuth <- authProvider.getLeoAuthToken
+
+      parsedUUID <- F.delay(Either.catchNonFatal(UUID.fromString(params.billingProfileId.value)))
+      profileAttempt <- parsedUUID.traverse { uuid =>
+        bpmClient.getProfile(leoAuth, uuid)
+      }
+
+      maybeLimits = profileAttempt match {
+        case Right(Some(profile)) =>
+          profile.getOrganization.getLimits.asScala
+            .get("concurrentjoblimit")
+            .map(v => raw"config.concurrentJobLimit=${v}")
+        case _ => None
+      }
```

Comment:
after poking at this a bit, you could probably reduce this some:
```scala
      maybeLimits = profileAttempt.toOption.flatten.flatMap { profile =>
        profile.getOrganization.getLimits.asScala
          .get("concurrentjoblimit")
          .map(v => raw"config.concurrentJobLimit=${v}")
      }
```

Reply:
Done